docs/content/documentation/modules/phase_field/ICs/EBSD.md

```diff
@@ -0,0 +1,246 @@
+# Reading EBSD Data
+
+!image docs/media/phase_field/EBSDReader_example.png width=200px padding-left=20px float=right caption=Reconstructed microstructure using EBSDReader, created using the phase_field module example file `IN100-111grn.i`
+
+!image docs/media/phase_field/EBSDReader_stress.png width=200px padding-left=20px float=right clear=both caption=Reconstructed microstructure with stress, created using the combined module example file `EBSD_reconstruction_grain_growth_mech.i`
+
+<!--!slider max-width=75% left=220px
+     docs/media/phase_field/EBSDReader_example.png caption=Reconstructed microstructure using EBSDReader, created using the phase_field module example file `IN100-111grn.i`
+     docs/media/phase_field/EBSDReader_stress.png caption=Reconstructed microstructure with stress, created using the combined module example file `EBSD_reconstruction_grain_growth_mech.i`-->
+
+To read experimental [electron backscatter diffraction (EBSD)](https://en.wikipedia.org/wiki/Electron_backscatter_diffraction)
+data three components are needed
+
+* [`EBSDMesh`](phase_field/EBSDMesh.md) Mesh object
+* [`EBSDReader`](phase_field/EBSDReader.md) UserObjcet
+* Initial conditions (such as the `ReconVarIC` action provides)
+
+## Mesh
+
+The mesh is generated from the EBSD information in the specified EBSD data file
+to get an optimal reconstruction of the data. This is accomplished in the mesh
+block using the [`EBSDMesh`](phase_field/EBSDMesh.md) type. The mesh is created
+with one node per data point in the EBSD data file. If you wish to use mesh
+adaptivity and allow the mesh to get coarser during the simulation, the
+`uniform_refine` parameter is used to set how many times the mesh can be coarsened.
+The block takes the form:
+
+```text
+[Mesh]
+  type = EBSDMesh
+  filename = IN100_128x128.txt
+  uniform_refine = 4
+[]
+```
+
+## EBSD Reader UserObject
+
+The user object reads in the data file, using the name supplied in the mesh block,
+and stores a data object with the local data at each material point as well as the
+average data about each grain. The block syntax is very simple:
+
+```text
+  [./ebsd]
```

Comment:
I think you should include the [UserObjects] block as well so people no where to put this.